Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -154,12 +154,14 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         manifestResources: ManifestResourceProvider,
         isManifestSandboxEnabled: Bool = true,
         cacheDir: AbsolutePath? = nil,
+        sdkRoot: AbsolutePath? = nil,
```

Comment:
This is already present in `ManifestResourceProvider`

Reply:
Ah true, I missed that. Thanks!